test/spec/RuntimeConfiguration.spec.ts

```diff
@@ -1,4 +1,7 @@
-import { expect, test } from 'vitest'
+import { randomUUID } from 'crypto'
+import { unlinkSync, writeFileSync } from 'fs'
+import { tmpdir } from 'os'
```

Comment:
I prefer these
```suggestion
import { randomUUID } from 'node:crypto'
import { unlinkSync, writeFileSync } from 'node:fs'
import { tmpdir } from 'node:os'
```

Reply:
to add later

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-node-protocol.md